### PR TITLE
Outline / document invoicing MVP processes

### PR DESCRIPTION
**Pull request by rub1e**
Outlines the process for setting up new invoicers and making sure the autocrat system works so that invoices are sent to dwyl, the invoicer, and xero

### CODE REVIEW

File: incoming-invoices.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Update the first question (name) with the invoicer's name **exactly as it appears in the `Contractor Name` field in the spreadsheet**. (screenshot)
+
+Their name will now be available in the Form dropdown, and will link up to their banking/contact details in the Sheet
+
+### Sending 
+
+`Autocrat` is configured to send & compile automatically, and pretty instantly, IN THEORY.
+
+For currently unknown reasons this doesn't always happen, so you may have to check the Sheet and trigger manually
```
Comment (iteles) [inline incoming-invoices.md]: Who does this? The finance person once someone tells them that their invoices aren't going through?

File: incoming-invoices.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+* Get banking and contact details from invoicer and input into spreadsheet to set them up (if they aren't already)
+* Add their name to the `name` question on the form (if they're not there already)
+* Ask invoicer to fill in the form at [invoice.dwyl.com](https://invoice.dwyl.com)
+* At this stage, `Autocrat` should take care of the rest as it's set up to run on form change
+  * The output will be an email with PDF invoice from `no-reply@dwyl.com` to the invoicer, copying in `finance@dwyl.com`, and sending a copy to `Xero`  
+  * If nothing happens, go to the spreadsheet and trigger it manually (instructions below)
+
+### Setting up new invoicer / amending existing details
+
+After being granted access to the *Finance (private)* folder, go to `Invoicing`, where you'll find a file called `dwyl invoicing (responses)`
```
[the remark this answers is not in the document]
Comment (rub1e) [reply incoming-invoices.md]: Full stops added at the end of sentences which aren't bullet points 😃

File: incoming-invoices.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+![screenshot 2018-01-16 at 11 37 23](https://user-images.githubusercontent.com/11595920/34987222-525a3f14-fab2-11e7-866b-6a1cd6adac09.png)
+
+Go to `sheet 2` and fill in the invoicer's details there - banking, contact, etc. (screenshot)
```
[the remark this answers is not in the document]
Comment (rub1e) [reply incoming-invoices.md]: Not sure I understand the question. Autocrat is triggered when a form is submitted, takes the details from the form (name/hours/etc.), applies the matching details from the invoicer tab (rate/email/etc.), and sends the email out. 

So if you change an existing invoicer's name, you'd have to update their name on the Google Form.

But in your scenario where e.g. they get a pay bump so their rate goes up, all you need to do is change the rate - Autocrat won't run unless there's a job for it to run on (i.e. a form submission)*

*(Technically it probably gets triggered whenever you make any change to the spreadsheet at all, but unless there's a new job (form submission) for it to sort, nothing will happen.)

Is that what you're asking about? I've made a few more edits so hopefully that's more clear now

File: incoming-invoices.md (hunk 1 above)
Comment (rub1e) [reply incoming-invoices.md]: I've added a note at the end of the form to say that if you haven't received an email within an hour, contact dwyl finance. This way we should get warning if autocrat is being temperamental

But there's really no way to be sure, so I've suggested that a weekly check is done to ensure there are no rows waiting to be processed.

**Pull request merged**